index.html

```diff
@@ -21,33 +21,122 @@ <h2 class="notoc">Contents</h2>
 
 
 
-
-
+<section id="when_should_review_be_requested">
+  <h2>When should wide review be requested?</h2>
+  <p>A document is available for review from the moment it is first <a
+  href='https://www.w3.org/Consortium/Process/#publishing'>published</a>.
+  Working Groups should <a
+  href='https://www.w3.org/Consortium/Process/#formally-addressed'>formally
+  address</a> any substantive review comment about a technical report in a
+  timely manner.</p>
+  <p>Wide review should or must be requested:</p>
+  <ul>
+  <li>After a <a href="https://www.w3.org/Consortium/Process/#first-wd">First
+  Public Working Draft</a> is published (for most documents).
+  <p>Working Groups are often reluctant to make substantive changes to a mature
+  document, so reviewers should get a chance to send substantive technical
+  reviews as early as possible. This is especially important for <a
+  href="#how_to_get_horizontal_review">horizontal reviews</a>.</p>
+  </li>
+  <li>Before a document gets advanced to <a
+  href="https://www.w3.org/Consortium/Process/#transition-cr">Candidate
+  Recommendation</a> or gets updated as a <a
```

Comment:
I suggest a more precise definition of "before", such as "maximum 6 months before entering CR" or something. The time has to be reasonable to receive and process comments, but not so long that wide reviews might have gone stale yet could be considered by the definition as acceptable.

Reply:
A little more guidance would be helpful, but a specific time period is maybe not a good idea. The concern is the amount of churn: if the document has only had editorial improvements and minor fixes in those six months, there should be no problem with the horizontal reviews. (This happens reasonably often in the CSSWG, because there are so many specs and few editors, things swap in and out of attention; and the final stages of polishing before CR are often subtle algorithm tweaks that wouldn't impact HR.)